test/controllers/graphql_controller_v2_test.rb

```diff
@@ -398,9 +400,9 @@ def setup
     end
   end
 
-  test 'graphqlv2 geobox search with geodistance search' do
+  test 'graphql geobox search with geodistance search' do
```

Comment:
Same question for this test as the other geospatial tests - should we be adding the source parameter here?

Reply:
We shouldn't need to in this case because our assertions don't care if there are more than 10,000 results.